src/main/java/techreborn/init/TRDispenserBehavior.java

```diff
@@ -121,5 +123,27 @@ public ItemStack dispenseSilently(BlockPointer pointer, ItemStack stack) {
 				}
 			}
 		});
+
+		for (ModFluids fluid : ModFluids.values()) {
+			DispenserBlock.registerBehavior(fluid, new ItemDispenserBehavior() {
+				public ItemStack dispenseSilently(BlockPointer pointer, ItemStack stack) {
+					RebornBucketItem bucket = (RebornBucketItem) stack.getItem();
+					BlockPos blockPos = pointer.getPos().offset(pointer.getBlockState().get(DispenserBlock.FACING));
+
+					if (bucket.placeFluid(null, pointer.getWorld(), blockPos, null)) {
+						ItemStack emptyCell = new ItemStack(Items.BUCKET);
```

Comment:
Nit: emptyBucket

Reply:
I have pushed a new commit renaming this.  Thank you!